platformLibs/src/platform/osx/darwin.def

```diff
@@ -33,4 +33,4 @@ headerFilter = **
 excludedFunctions = __tg_promote
 
 compilerOpts = -D_XOPEN_SOURCE -DSHARED_LIBBIND -D_DARWIN_NO_64_BIT_INODE -DSYSCTL_DEF_ENABLED
-linkerOpts = -ldl -lz
+linkerOpts = -ldl -lz -lcurses -undefined dynamic_lookup
```

Comment:
This affects linkage of all libraries and Kotlin code.
I believe we should find better solution if possible.
Would supporting regexp in `excludedFunctions` help?

Reply:
Yeah, but maybe not in this PR, but in the followup where we'll use static libs.

Reply:
Actually, after adding `bz2`, `compression`, `ate` and `bsm` libs only a couple of dozens of functions remain unresolved and they can be explicitly added to `excludedFunctions`.